scripts/generate_all_protocols.py

```diff
@@ -427,18 +488,22 @@ def main() -> None:
         for package_path in all_protocols:
             log("=" * 100)
             log(f"Processing protocol at path {package_path}")
-            _bump_protocol_specification_id_if_needed(package_path)
-            _process_packages_protocol(package_path)
+            if not no_bump:
+                _bump_protocol_specification_id_if_needed(package_path)
+            # no_bump implies to ignore the docstring:
+            #  'It was created with protocol buffer compiler ... and aea version ...'
```

Comment:
Why are the two related?

Reply:
![image](https://user-images.githubusercontent.com/9467617/117949134-17804e80-b312-11eb-8d39-d3329ba3431e.png)

Reply:
if `--no-bump` is provided, then we don't expect the docstring to change due to just using an `aea` with a different version.
if `--no-bump` is not provided, then the script works as usual, taking the output of the generator as-is (after some post-processing that would happen anyway).

Reply:
? still doesn't answer my question. The protocol spec is specified in README.md. So I don't understand why this would be bumped in this case...

Reply:
Might you be confusing protocol id and protocol spec id @marcofavorito ?

Reply:
Probably no harm to leave it as you have done, but I don't think the protocol spec id would be bumped anyway unless manually bumped.

Reply:
`protocol_id` is bumped via another script